README.md

```diff
@@ -36,11 +39,12 @@ jobs:
     - name: Copy debian directory
       run: cp -a packaging/debian .
 
-    - uses: andy5995/gh-action-build-deb@v1
+    - uses: andy5995/gh-action-build-deb@v1.1.0
```

Comment:
This can be left as 'v1'. I'll push a v1 tag that will point to v1.1.0 when I push the tag ( or the versioning workflow will do it automatically if it's not too outdated ;)